examples/shapes_and_collections/hatch_style_reference.py

```diff
@@ -0,0 +1,40 @@
+"""
+=====================
+Hatch style reference
+=====================
+
+Hatching (pattern filled polygons) is supported currently in the PS,
+PDF, SVG and Agg backends only.
+"""
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+
+fig, axs = plt.subplots(2, 5, constrained_layout=True, figsize=(10, 5))
+
+hatches = ['/', '\\', '|', '-', '+', 'x', 'o', 'O', '.', '*']
+
+
+def hatches_plot(ax, h):
+    ax.add_patch(Rectangle((0, 0), 2, 2, fill=False, hatch=h))
+    ax.set_title(f"' {h} '")
```

Comment:
You were right that it's better the other way (underneath)

Reply:
I moved the title below the plots. 
![download](https://user-images.githubusercontent.com/11200528/87346640-897fdf80-c517-11ea-95b1-76cd402bc873.png)